sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor.cs

```diff
@@ -2283,6 +2253,64 @@ private async Task ValidateStorageConnectionAsync(CancellationToken cancellation
         ///
         private static int CalculateMaximumAdvisedOwnedPartitions() => (Environment.ProcessorCount * 2);
 
+        /// <summary>
+        /// Creates, starts, and enriches processing diagnostics scope in case batch tracing is enabled.
+        /// </summary>
+        /// <param name="eventBatch">A collection of <see cref="EventData"/> which is being processed.</param>
+        /// <returns>An instance of scope if tracing is enabled, null otherwise.</returns>
```

Comment:
```suggestion
        /// <summary>
        ///   Creates, starts, and enriches s processing diagnostics scope in case batch tracing is enabled.
        /// </summary>
        ///
        /// <param name="eventBatch">A collection of <see cref="EventData"/> which is being processed.</param>
        ///
        /// <returns>An instance of <see cref="DiagnosticScope" />, if tracing is enabled; otherwise, <c>null</c>.</returns>
        ///
```